### PR TITLE
Add basic ui

### PR DESCRIPTION
**Pull request by Kimutai01**
-Added UI and interactivity

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+      book_id = gets.chomp.to_i
+      puts 'Enter rental date (YYYY-MM-DD):'
+      date = gets.chomp
+      @rentals << Rental.new(date, @people[person_id], @books[book_id])
+      puts "\n"
+      puts 'Rental created!'
+    else
+      puts 'You do not have permission to rent books!'
+    end
+  end
```
Comment (ahmetbozaci) [inline app.rb]: - [ ] When creating a rental ı can't see the index number(inside bracket) next to the name. Kindly add it.

image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+        list_people
+      when '3'
+        create_person
+      when '4'
+        create_book
+      when '5'
+        create_rental
+      when '6'
+        list_rentals_by_person_id
+      else
```
Comment (ahmetbozaci) [inline app.rb]: - [ ] I cant see rentals correctly. Kindly fix it.

image

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+  end
+
+  def run
+    loop do
+      option = show_menu
+      case option
+      when '1'
+        list_books
+      when '2'
+        list_people
```
Comment (ahmetbozaci) [inline app.rb]: - [ ] Kindly add an index next to the persons and books.
[video](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
image

**Review by ahmetbozaci — CHANGES_REQUESTED** [review]
# ♻️♻️♻️ STATUS: Changes Required  ♻️♻️♻️


 @Kimutai01  ✋🏼 

#### Your project is great. You are almost done. However, there are some issues you should fix. Please fix them then your project will be approved.

___ 

### 💯  Highlights 💯
- Your git-flow is correct. ✔️
- You set up linter files and no errors. ✔️
- Your pull request is very professional. It has a meaningful title and summary. Keep it up.✔️
- I can create books, people, and rentals without errors
___


### ♻️ Request changes ♻️

- [ ] _Check the comments under the review._

___

### Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._
___

### Cheers and Happy coding! 💻 👏
almost


___

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification._

_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._


___

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_

[GitHub](https://github.com/ahmetbozaci) | [LinkedIn](https://linkedin.com/in/ahmetbozaci) | [Twitter](https://twitter.com/ahmtbozaci)

_Reviewed by [Ahmet Bozacı ](https://github.com/ahmetbozaci)_

**Review by Lordkaito — APPROVED** [review]
Hi @Kimutai01,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**